Makefile

```diff
@@ -15,8 +15,10 @@ GOBUILD := GO111MODULE=on go build -v
 GOINSTALL := GO111MODULE=on go install -v
 GOMOD := GO111MODULE=on go mod
 
-COMMIT := $(shell git describe --abbrev=40 --dirty | sed -E 's/.*-g([0-9a-f]{40})(-dirty)?/\1\2/')
-LDFLAGS := -ldflags "-X $(PKG).Commit=$(COMMIT)"
+COMMIT := $(shell git describe --abbrev=40 --dirty --tags)
+COMMIT_HASH := $(shell git rev-parse HEAD)
+DIRTY := $(shell git diff-index --quiet HEAD -- || echo -dirty)
```

Comment:
Instead of `-dirty` we should store it as `dirty`.

Reply:
Ah good idea! Addressed with the latest push :).